apps/editing-toolkit/editing-toolkit-plugin/full-site-editing-plugin.php

```diff
@@ -338,6 +338,10 @@ function load_tags_education() {
  * (Core Full Site Editing)
  */
 function load_wpcom_site_editor() {
+	// This is no longer needed after Gutenberg 12.1 due to the Navigation menu no longer being inscrutable.
```

Comment:
Is this comment necessary? I feel that people can always refer to the `git blame`, check the PR and the explanation on why it is not needed anymore. Comments are often misleading.

Reply:
The point of this code comment is to encourage future maintainers to investigate if this can be removed, should I not get to it. But your point is a fair one, given the number of years-later `@todo` and `@fixme` comments strewn about.

Reply:
How about an issue instead of a comment? You could even link it as a comment inside the code.

Reply:
I think its a handy companion to the version check below. As long as we are conditionally enabling functionality based off a magic gutenberg version number, it seems good to note why for easy reference.

Hopefully it wont get outdated or misleading, as it is accurate to the code beneath it. When that code is updated, then the comment should be as well. Ideally, once we don't support any pre-12.2 Gutenberg version on dotcom anymore we can just remove this entire site editor section of this plugin.

Reply:
Thanks for creating that issue @Addison-Stavlo ! I intended to do the same but this has been the week of inability to maintain an attention span.